mission_7/queries.sql

```diff
@@ -0,0 +1,111 @@
+/*
+  다음 경우들에 대해 총 14개의 SQL 쿼리를 작성해 주세요.
+  예시로 값이 필요한 경우 적당한 값으로 채워넣어서 작성하면 됩니다. 
+*/
+
+/*
+  1. 내 정보 업데이트 하기
+  - 닉네임을 "test"로 업데이트
+  - 현재 로그인한 유저 id가 1이라고 가정
+*/
+UPDATE users SET name = 'test' WHERE id = 1;
+
+
+/*
+  2. 내가 생성한 상품 조회
+  - 현재 로그인한 유저 id가 1이라고 가정
+  - 최신 순으로 정렬
+  - 10개씩 페이지네이션, 3번째 페이지
+*/
+SELECT * FROM products WHERE user_id = 1 ORDER BY id DESC LIMIT 10 OFFSET 20;
```

Comment:
시간 정렬을 할 때, 생성일 기준 + 동률 대비 PK를 보조로 사용하시는 것을 권장드립니다.

```suggestion
SELECT * FROM products WHERE user_id = 1 ORDER BY createdAt DESC, id DESC LIMIT 10 OFFSET 20;
```